resources/lib/main_module.py

```diff
@@ -227,7 +227,8 @@ def setforcedview(self):
     def get_youtube_listing(searchquery):
         '''get items from youtube plugin by query'''
         lib_path = u"plugin://plugin.video.youtube/kodion/search/query/?q=%s" % searchquery
-        return KodiDb().files(lib_path)
+        metadatautils = MetadataUtils()
+        return metadatautils.kodidb.files(lib_path)
 
```

Comment:
please close the metadadatautils object after initialising it, otherwise warnings will be printed in the logs about garbage not being cleaned up.

so: 

metadatautils = MetadataUtils()
files = metadatautils.kodidb.files(lib_path)
del metadatautils
return files

Reply:
Changed, next time better you change mine code.
You always know what is best....I just try to add/fix something but don't have knowledge to finished up.
Tnx :)